### PR TITLE
Add Mobile Design for Transferees Table for SRWs and Covenants

### PR DESCRIPTION
**Pull request by Abradat**

### CODE REVIEW

File: portal-frontend/src/app/features/notifications/edit-submission/transferees/transferees.component.ts [hunk 1]
```diff
@@ -22,12 +23,15 @@ export class TransfereesComponent extends StepComponent implements OnInit, OnDes
   displayedColumns: string[] = ['type', 'fullName', 'organizationName', 'phone', 'email', 'actions'];
 
   private submissionUuid = '';
+  isMobile = false;
+  VISIBLE_COUNT = 5;
```
Comment (trslater) [inline portal-frontend/src/app/features/notifications/edit-submission/transferees/transferees.component.ts:27]: As a constant, I would move this outside of the component and define it with `const`. I'd also change the name to something like `VISIBLE_COUNT_INCREMENT`.

Comment (Abradat) [reply portal-frontend/src/app/features/notifications/edit-submission/transferees/transferees.component.ts]: Moved and renamed the constants to constants.ts file in shared module. (Same as alcs-frontend)

Comment (trslater) [reply portal-frontend/src/app/features/notifications/edit-submission/transferees/transferees.component.ts]: Nice. This is even better.

**Review by trslater — APPROVED** [review]
LGTM 🚀

**Pull request merged**